### DISCUSSION TITLE
Object's Scene Synced to Clients

### DISCUSSION THREAD

**Discussion by maxkratt** [FirstGearGames/FishNet#680] [Feature Request]
It's come up quite a bit in the Discord that the scene a NetworkObject spawns in on the server isn't synced to clients, instead FishNet will instantiate the object in the active scene on the client. While this makes sense most of the time and saves bandwidth, there are times when it would be helpful to have the scene synced. This can be done with a script on the object syncing the scene using the payload or a SyncVar, but it might be more helpful for users if there was a way to do so automatically.

**Comment by FirstGearGames** [comment]
This is already planned. Not only is this already planned this is the incomplete commit that accidentally got merged into 4.3.3 causing the break. Once the new tech we're working on is ready this is going in.

**Comment by FirstGearGames** [comment]
Still planned but also still requires a refactor.

**Comment by FirstGearGames** [comment]
Added to roadmap in V5.
